packages/ui-extensions-server-kit/src/ExtensionServerClient/types.ts

```diff
@@ -230,7 +230,7 @@ declare global {
   }
 }
 
-export const AVAILABLE_SURFACES = ['admin', 'checkout', 'post-checkout', 'pos', 'customer-accounts'] as const
+export const AVAILABLE_SURFACES = ['admin', 'checkout', 'post-checkout', 'pos', 'customer-accounts', null] as const
```

Comment:
This change makes me wonder if we are overloading the concept of `ui extensions`.  `WebPixel` and `Theme` extensions end up in the dev console, but are very different beasts to everything else in this list.  They have no surface, hence the additional null `value`.

Reply:
Can they be interacted with through the console? Of is it only to show that they are being served?

Reply:
Thinking about this some more, with the Dev Index project the extensions dev console is about to become more than "just" extensions, so I think this is ok.

Theres a bigger change on the horizon though, which is to remove the "extensions" namespace from the dev console and supporting code.